service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/AzureDatabaseUtilsRunner.java

```diff
@@ -0,0 +1,382 @@
+package bio.terra.workspace.service.resource.controlled.cloud.azure.database;
+
+import static bio.terra.workspace.service.resource.controlled.cloud.azure.AzureUtils.getResourceName;
+
+import bio.terra.common.iam.BearerToken;
+import bio.terra.stairway.exception.RetryException;
+import bio.terra.workspace.amalgam.landingzone.azure.LandingZoneApiDispatch;
+import bio.terra.workspace.app.configuration.external.AzureConfiguration;
+import bio.terra.workspace.common.utils.RetryUtils;
+import bio.terra.workspace.service.iam.SamService;
+import bio.terra.workspace.service.resource.controlled.cloud.azure.KubernetesClientProvider;
+import bio.terra.workspace.service.workspace.WorkspaceService;
+import bio.terra.workspace.service.workspace.model.AzureCloudContext;
+import io.kubernetes.client.PodLogs;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.apis.CoreV1Api;
+import io.kubernetes.client.openapi.models.V1Container;
+import io.kubernetes.client.openapi.models.V1EnvVar;
+import io.kubernetes.client.openapi.models.V1ObjectMeta;
+import io.kubernetes.client.openapi.models.V1Pod;
+import io.kubernetes.client.openapi.models.V1PodSpec;
+import io.kubernetes.client.openapi.models.V1PodStatus;
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+
+/**
+ * Wrapper around <a href="{@docRoot}/azureDatabaseUtils/README.md">AzureDatabaseUtils</a>. Each
+ * public method in this class is a wrapper around a command in AzureDatabaseUtils. Every call
+ * creates a pod in the default namespace
```

Comment:
I am still not clear what a pod is.

Reply:
That is, I am familiar with a kubernetes pod, but why do each of these calls create a pod?

Reply:
Think of a pod as a VM that runs a collection of containers. But the containers and their configuration are not mutable. So each time we want to run a different container specification to run, we need a new pod.

Alternatively we could have a single pod running a generic container that we connect to to run commands. But that is more complicated IMO and would continuously cost money.

Reply:
> Alternatively we could have a single pod running a generic container that we connect to to run commands. But that is more complicated IMO...

That makes a lot of this make more sense, in terms of the why.